adapter/exporter/src/main/java/org/mobilitydata/gtfsvalidator/exporter/ProtobufNoticeExporter.java

```diff
@@ -786,6 +786,21 @@ public void export(final StopTimeArrivalTimeAfterDepartureTimeNotice toExport) t
                 .writeTo(streamGenerator.getStream());
     }
 
+    @Override
+    public void export(final StopTooFarFromTripShape toExport) throws IOException {
+        // TODO - Needs review - Are these right?
```

Comment:
I'm not sure I got the protobuf export correct - please review!

Reply:
If we keep the current structure of `StopTooFarFromTripShape`, this misses the export of `KEY_COMPOSITE_KEY_THIRD_VALUE`, `KEY_COMPOSITE_KEY_THIRD_PART`, `KEY_COMPOSITE_KEY_FOURTH_VALUE`, `KEY_COMPOSITE_KEY_THIRD_PART`.

Reply:
Thanks! I added `KEY_COMPOSITE_KEY_THIRD_VALUE` and `KEY_COMPOSITE_KEY_THIRD_PART` based on other examples in the class, but I'm not sure what `protoBuilder` methods the `KEY_COMPOSITE_KEY_FOURTH_VALUE` and `KEY_COMPOSITE_KEY_FOURTH_PART` should map to. It looks like the other notices that use these same constants also don't export them to protobuf. Any suggestions?

Reply:
I'd say to pick `protoBuilder` methods that are not used

Reply:
I thought about that, but seems like there are two problems with that approach:
1. It seems like that's abusing the protobuf model. How would another application using the same .proto file know that a particular value was written to a random field?
1. There don't seem to be any remaining String or int methods I could write to (unless I'm missing one?)

Seems like for composite keys the .proto file should really be changed so that there is `OtherCsvKey1Name`, `OtherCsvKey2Name`, `OtherCsvKey3Name`, etc. fields. Or maybe better yet change to using fields named `CompositeKeyPart1`, `CompositeKeyPart2`, etc.

Reply:
1. that is question that has been raised during one of the monthly meetings, and if I recall correctly there was no clear instruction regarding this. However, we had the approval to add new error types to `gtfs_validation.proto`. I guess that we should do the same for the `protoBuilder` methods to make clear which `protoBuilder` method is used to write a particular field.

Reply:
2. We could use these methods: 
```                .setAltEntityValue()
                .setEntityName()
                .setValue()
                .setParentEntityName()``` to export the rest of the information for now. And then see for adding more fields to the `.proto` file and refactoring the existing code to use new generated methods.

Reply:
>I guess that we should adopt the same approach for the protoBuilder fields, to make clear which protoBuilder method is used to write a particular field.

Yes, I believe the `protoBuilder` should be autogenerated again after the .proto file is updated, which will give us the getters/setters for those fields.

Given that this is an issue beyond just the notice in this PR, I'd prefer to put off implementation of this for another PR as the changes will touch a lot of files. I've opened an issue here to track - https://github.com/MobilityData/gtfs-validator/issues/364.

Reply:
Thanks